docs/pages/database-access/auto-user-provisioning/database-object-permissions.mdx

```diff
@@ -0,0 +1,205 @@
+---
+title: Database object permissions
+description: Understand the concepts behind database object permissions managed by Teleport.
+---
+
+## Overview
+
+Database object permissions is a Teleport feature that enables the configuration of access levels for individual users to specific database objects based on their required level of access.
+
+Database objects are labeled based on import rules, and these labels correspond to specific permissions configured within user roles.
+
+Permissions are granted for the duration of a connection and are automatically revoked upon its closure.
+
+## Database object import rules
+
+A database object import rule in Teleport is a resource that defines the labels to be applied to database objects imported into Teleport. If a specific object does not match any of the rules, it will not be imported.
+
+By default, if no import rules are present, Teleport will automatically create the `import_all_objects` rule on startup:
+
+```yaml
+kind: db_object_import_rule
+metadata:
+  name: import_all_objects
+spec:
+  # Priority determines how important the rule is, with lower number indicating lower priority.
+  # In case of conflicts, when the same label is applied by two rules, 
+  # the label applied by rule with higher priority wins.
+  priority: 0
+  # database_labels is a filter specifiying which database resources are in scope of this rule.
+  database_labels:
+  - name: '*'
+    values:
+    - '*'
+  # Each mapping, if matched, introduces a set of labels applied to database object.
+  # Database objects without labels are not imported.
+  mappings:
+  - add_labels:
+      database: '{{obj.database}}'
```

Comment:
Do we have plans to add a reference table of the data fields (and functions?) available in `mappings`? We've got examples of this in the [Role Reference](https://goteleport.com/docs/access-controls/reference/#template-expressions-for-access-to-infrastructure-resources).